src/sentry/rules/conditions/high_priority_issue.py

```diff
@@ -17,8 +17,8 @@ class HighPriorityIssueCondition(EventCondition):
     id = "sentry.rules.conditions.high_priority_issue.HighPriorityIssueCondition"
     label = "Sentry marks an issue as high priority"
 
-    def is_high_severity(self, state: EventState, group: Optional[Group]) -> bool:
-        if not group:
+    def is_new_high_severity(self, state: EventState, group: Optional[Group]) -> bool:
+        if not group or not state.is_new:
```

Comment:
@snigdhas is `is_new` only true on the first event?

Reply:
I believe so - it's only set to True if a GroupHash is created, so it's the first event for a new group